build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -78,19 +89,45 @@ for i in $(ls ${ROOT_DIR}/models/tflite/); do
     --iree-vm-emit-polyglot-zip=false \
    "${IREE_MODEL_DIR}/${MODEL_NAME}.mlir" \
     --o "${IREE_MODEL_DIR}/cuda/${MODEL_NAME}.vmfb" || true
+
+  echo "Compiling ${MODEL_NAME}_fp16.vmfb for cuda..."
+  "${IREE_COMPILE_PATH}" \
+    --iree-input-type=tosa \
+    --iree-hal-target-backends=cuda \
+    --iree-hal-cuda-llvm-target-arch=sm_80 \
+    --iree-flow-demote-f32-to-f16 \
+    --iree-llvm-debug-symbols=false \
+    --iree-vm-bytecode-module-strip-source-map=true \
+    --iree-vm-emit-polyglot-zip=false \
+   "${IREE_MODEL_DIR}/${MODEL_NAME}.mlir" \
+    --o "${IREE_MODEL_DIR}/cuda/${MODEL_NAME}_fp16.vmfb" || true
```

Comment:
Same here. Do we expect the compilation will fail?

Reply:
Yes, there is a high failure rate unfortunately e.g. quantized models tend to not compile for CUDA, some compile flags such as im2col do not compile for many models, etc.

The script compiles and benchmarks as a best effort.